stable/restore/README.md

```diff
@@ -162,45 +162,18 @@ The following tables list the configurable parameters of the backup chart and th
 
 | Parameter | Description | Default |
 | ----- | ----------- | ------ |
-| `timeout` | Job deadline (timeout) | 1800 |
-| `backupPvc` | PVC name for the backup volume | `nil` |
-| `archivePvc` | PVC name for the archive volume | `nil` |
-| `backupSet` | The backup set name to restore from | `nil` |
+| `restore.target` | Where to restoe `TO` | `{{ .Values.database.name }}` |
+| `restore.source` | Where to retore `FROM` [ backupset | url | `:latest` ] | `:latest` |
```

Comment:
typo in `restore`

Reply:
*ugh*

I've fixed, and it will be in the next push.